toolkit/cli/commands/src/address_association_signatures.rs

```diff
@@ -0,0 +1,56 @@
+use anyhow::{anyhow, Ok};
+use byte_string::ByteString;
+use clap::Parser;
+use parity_scale_codec::Encode;
+use partner_chains_smart_contracts_commands::PaymentFilePath;
+use serde::Serialize;
+use serde_json::json;
+use sidechain_domain::*;
+use sp_address_associations::AddressAssociationSignedMessage;
+use std::str::FromStr;
+
+#[derive(Clone, Debug, Parser)]
+#[command(author, version, about, long_about = None)]
+pub struct AddressAssociationSignaturesCmd<
+	PartnerchainAddress: Clone + Sync + Send + FromStr + Encode + 'static,
```

Comment:
Maybe over-engineering, but this command could be parametrized by `PartnerchainsArgs` that is `clap::Parser` and `Into<T>`, where `T: Serialize + Encode + Clone` should be enough. But it will require adding a type in node-commands.